core/server/models/base/index.js

```diff
@@ -773,6 +773,8 @@ ghostBookshelf.Model = ghostBookshelf.Model.extend({
             model.hasTimestamps = false;
         }
 
+        options.require = true;
```

Comment:
Yeah i noticed that, i don't think it's a problem though - what do you think, i should omit it?

Reply:
hmmmmmm it can be weird when you debug a problem and you e.g.

put a console.log in the `onSaving` hook and you look at the options and you see `require:true` and then you wonder: why is it passing require:true?

It's not suuuper important, but could be confusing and is not 100% correct passing require:true to the save call?